backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java

```diff
@@ -929,27 +951,36 @@ IGitMacheteRepositorySnapshot discoverLayoutAndCreateSnapshot(int mostRecentlyCh
 
   @Getter
   @RequiredArgsConstructor(access = AccessLevel.PRIVATE)
-  private static final class CreatedAndSkippedBranches<T extends BaseManagedBranchSnapshot> {
+  private static final class CreatedAndSkippedAndDuplicatedBranches<T extends BaseManagedBranchSnapshot> {
```

Comment:
Nit/paranoia: since both duplicated and skipped branches have the same type, they can be confused in contexts where both are expected (like `GitMacheteRepositorySnapshot`'s c'tor). I'd suggest keeping one fixed order (alphabetical FTW) everywhere in names/fields/methods to reduce the risk of such mistake... so this would be `CreatedAndDuplicatedAndSkippedBranches`

Reply:
Whooops, good point. Firstly I added the duplicated field as the last one, then I moved it b4 skipped to keep the order but I forgot about the class name Xd